tutorials/k8s-fluentbit-observability/index.mdx

```diff
@@ -0,0 +1,233 @@
+
+This process will be done using [FluentBit](https://fluentbit.io/), a lightweight logs and metrics processor which acts as a gateway between containers and the Cockpit endpoints, when configured in a Kubernetes cluster.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
+  - You have [retrieved your Grafana credentials](/observability/cockpit/how-to/retrieve-grafana-credentials/)
+  - You have [created a Kapsule cluster](/containers/kubernetes/how-to/create-cluster/)
+  - You have set up [kubectl](/containers/kubernetes/how-to/connect-cluster-kubectl/) on your machine
+  - You have installed `helm`, the Kubernetes [package manager](https://helm.sh/), on your local machine (version 3.2+)
+</Message>
+
+## Configuring the Fluent Bit service
+
+Fluent Bit will be installed as a Helm package configured to target your Kubernetes resources as inputs and your Observability cockpit as an output.
+
+As the installation will be done through Helm, add the Helm repository for Fluent Bit to your machine:
+
+```bash
+helm repo add fluent https://fluent.github.io/helm-charts
+helm repo update
+```
+
+Then, create a [values file for Helm](https://helm.sh/docs/chart_template_guide/values_files/) named `values.yaml` that we will use to configure Fluent Bit.
+
+In the `values.yaml` file, create a first section `config.service` to configure the Fluent Bit master process:
+
+```yaml
+config:
+  service: |
+    [SERVICE]
+        Flush                 1
+        Log_level             info
+        Daemon                off
+        Parsers_File          custom_parsers.conf
+        HTTP_Server           on
+        HTTP_Listen           0.0.0.0
+        HTTP_PORT             2020
+```
+
+- `Flush 1`: Collects logs every second.
+- `Log_level info`: Displays informational logs in the Fluent Bit pods.
+- `Daemon off`: Run Fluent Bit as the foreground process in its pods.
+- `Parsers_File custom_parsers.conf`: Loads additional log parsers that we will define later on.
+- `HTTP_Server on`: Enables Fluent Bit's built-in HTTP server.
+- `HTTP_Listen 0.0.0.0`: Listen on all interfaces exposed by your pod.
+- `HTTP_PORT 2020`: Listen to port 2020.
+
+<Message type="note">
+    Fluent Bit's HTTP server needs to be enabled in order to communicate with your Cockpit.
+</Message>
+
+## Configuring observability inputs
+
+We will configure Fluent Bit to retrieve the metrics (CPU, memory, disk usage...) from your Kubernetes nodes and the applicative logs from your running pods.
+
+In the `values.yaml` file, create a new section `config.inputs`:
+
+```yaml
+  inputs: |
+    [INPUT]
+        Name                  node_exporter_metrics
+        Tag                   node_metrics
+        Scrape_interval       2
```

Comment:
Can you update the scrape_interval to be `60s` ? 
A `2s` scrape interval will increase the cost a lot on Cockpit.
Having it to `60s` divide the price by 30.